numpy/core/src/umath/loops_arithmetic.dispatch.c.src

```diff
@@ -0,0 +1,132 @@
+/*@targets
+ ** $maxopt baseline
+ ** sse2 sse41 avx2 avx512_skx
+ ** vsx2
+ ** neon
+ **/
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#include "simd/simd.h"
+#include "loops_utils.h"
+#include "loops.h"
+#include "lowlevel_strided_loops.h"
+#include<signal.h> 
+// Provides the various *_LOOP macros
+#include "fast_loop_macros.h"
+
+//###############################################################################
+//## Unsigned Integers
+//###############################################################################
+/********************************************************************************
+ ** Defining the SIMD kernels
+ ********************************************************************************/
+#ifdef NPY_SIMD
```

Comment:
```suggestion
#if NPY_SIMD
```
`NPY_SIMD` is a numeric definition returns the size of SIMD bits if there's an supported SIMD extension in the scope otherwise it returns zero.

Reply:
ping

Reply:
Apologies for missing this.